drivers/rtc/rtc_rv8263.c

```diff
@@ -0,0 +1,701 @@
+#endif
+
+#if CONFIG_RTC_UPDATE && DT_ANY_INST_HAS_PROP_STATUS_OKAY(int_gpios)
+int rv8263_update_callback(const struct device *dev, rtc_update_callback callback, void *p_user_data)
+{
+    struct rv8263c8_data *const data = dev->data;
+
+    if (dev == NULL) {
+        return -EINVAL;
+    }
+
+    (void)k_sem_take(&data->lock, K_FOREVER);
+    data->update_cb = callback;
+    data->update_cb_data = p_user_data;
+    k_sem_give(&data->lock);
+
+    return rv8263c8_update_enable_timer(dev);
+}
+#endif
+
+#ifdef CONFIG_RTC_CALIBRATION
+int rv8263c8_calibration_set(const struct device *dev, int32_t calibration)
+{
+    int err;
+    int8_t reg;
+    struct rv8263c8_data *const data = dev->data;
+    const struct rv8263c8_config *config = dev->config;
+
+    if (dev == NULL) {
+        return -EINVAL;
+    }
+
+    // NOTE: The RTC API is using a PPB (Parts Per Billion) value. The RTC is using PPM.
+
+    return 0;
+}
+
+int rv8263c8_calibration_get(const struct device *dev, int32_t *calibration)
+{
+    int err;
+    uint8_t value;
+    struct rv8263c8_data *const data = dev->data;
+    const struct rv8263c8_config *config = dev->config;
+
+    if ((dev == NULL) || (calibration ==  NULL)) {
+        return -EINVAL;
+    }
+
+    return 0;
+}
+#endif
+
+static const struct rtc_driver_api rv8263c8_driver_api = {
+    .set_time = rv8263c8_time_set,
+    .get_time = rv8263c8_time_get,
+#if CONFIG_RTC_ALARM
+    .alarm_get_supported_fields = rv8263c8_alarm_get_supported_fields,
+    .alarm_set_time = rv8263c8_alarm_set_time,
+    .alarm_get_time = rv8263c8_alarm_get_time,
+    .alarm_is_pending = rv8263c8_alarm_is_pending,
+#if DT_ANY_INST_HAS_PROP_STATUS_OKAY(int_gpios)
+    .alarm_set_callback = rv8263c8_alarm_set_callback,
+#endif
+#endif
+#if CONFIG_RTC_UPDATE && DT_ANY_INST_HAS_PROP_STATUS_OKAY(int_gpios)
+    .update_set_callback = rv8263_update_callback,
+#endif
+#ifdef CONFIG_RTC_CALIBRATION
+    .set_calibration = rv8263c8_calibration_set,
+    .get_calibration = rv8263c8_calibration_get,
+#endif
+};
+
+#define RV8263_DEFINE(inst)                                                         \
+    static struct rv8263c8_data rv8263c8_data_##inst;                               \
+    static const struct rv8263c8_config rv8263c8_config_##inst = {                  \
+        .i2c_bus = I2C_DT_SPEC_INST_GET(inst),                                      \
+        .clkout = DT_INST_PROP(inst, clkout),                                       \
+        IF_ENABLED(DT_ANY_INST_HAS_PROP_STATUS_OKAY(int_gpios),                     \
+            (.int_gpio = GPIO_DT_SPEC_INST_GET_OR(inst, int_gpios, { 0 }),))        \
+    };                                                                              \
+    DEVICE_DT_INST_DEFINE(inst, &rv8263c8_init, NULL, &rv8263c8_data_##inst,        \
+                  &rv8263c8_config_##inst, POST_KERNEL, CONFIG_RTC_INIT_PRIORITY,   \
+                  &rv8263c8_driver_api);
+
+DT_INST_FOREACH_STATUS_OKAY(RV8263_DEFINE)
```

Comment:
You need to add a newline at the end of the file, this is true for the Kconfig file as well. In the IDE you are using, it should look like
```
699 |                   &rv8263c8_driver_api);
700 | DT_INST_FOREACH_STATUS_OKAY(RV8263_DEFINE)
701 |
```
instead of
```
699 |                   &rv8263c8_driver_api);
700 | DT_INST_FOREACH_STATUS_OKAY(RV8263_DEFINE)
```